vdr/didweb/store.go

```diff
@@ -120,3 +151,43 @@ func (s *sqlStore) list() ([]did.DID, error) {
 	}
 	return result, nil
 }
+
+// createService creates a new service in the DID document identified by subjectDID.
+// It does not validate the service.
+func (s *sqlStore) createService(subjectDID did.DID, service did.Service) error {
```

Comment:
For all DB related operations, make sure to deal with:
- `ErrRecordNotFound`
- `ErrDuplicatedKey`
- (`ErrForeignKeyViolated`)

Otherwise 500 errors will be generated where a 400 should be generated.